benchmarks/lobsters/benchmark.rb

```diff
@@ -0,0 +1,44 @@
+require 'harness'
+
+ENV['RAILS_ENV'] ||= 'production'
+ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK'] = '1' # Benchmarks don't really have 'production', so trash it at will.
+
+Dir.chdir __dir__
+use_gemfile extra_setup_cmd: "bin/rails db:drop db:create && sqlite3 db/production.sqlite3 < db/faked_bench_data.sql"
+
+require_relative 'config/environment'
+require_relative "route_generator"
+
+app = Rails.application
+generator = RouteGenerator.new(app)
+generator.do_login # We want to be logged in for all generated requests
```

Comment:
There were a couple of things that were significant different when not logged in (e.g. checking if the user had warnings for too many flags on their posts) that I wound up removing because the code was too hard to port to SQLite. But yeah, mostly I checked only when logged in because that turns off the cache on most places.

There are a couple of places that still use the cache, but there's no obvious way to turn those off except switching to a Null cache, and I thought that might be more disruptive.

It would be fine to check non-logged-in, but yeah, we'd want to deactivate the cache in a different way and I don't think it would be significantly different in performance.

Reply:
If we ever *do* expand to more of the full life cycle (add more POSTs, create and delete stories and comments, etc.) then it'll make more sense to do that because we'll be tracking a lot of independent users in the benchmark anyway. It seemed like extra complexity for no real reason in the current setup.